keyboards/gray_studio/think65v3/info.json

```diff
@@ -0,0 +1,266 @@
+                {"label": "k1d", "matrix": [1, 13], "x": 12.5, "y": 1},
+                {"label": "k1e", "matrix": [1, 14], "x": 13.5, "y": 1, "w": 1.5},
+                {"label": "k1f", "matrix": [1, 15], "x": 15, "y": 1},
+
+                {"label": "k20", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
+                {"label": "k22", "matrix": [2, 2], "x": 1.75, "y": 2},
+                {"label": "k23", "matrix": [2, 3], "x": 2.75, "y": 2},
+                {"label": "k24", "matrix": [2, 4], "x": 3.75, "y": 2},
+                {"label": "k25", "matrix": [2, 5], "x": 4.75, "y": 2},
+                {"label": "k26", "matrix": [2, 6], "x": 5.75, "y": 2},
+                {"label": "k27", "matrix": [2, 7], "x": 6.75, "y": 2},
+                {"label": "k28", "matrix": [2, 8], "x": 7.75, "y": 2},
+                {"label": "k29", "matrix": [2, 9], "x": 8.75, "y": 2},
+                {"label": "k2a", "matrix": [2, 10], "x": 9.75, "y": 2},
+                {"label": "k2b", "matrix": [2, 11], "x": 10.75, "y": 2},
+                {"label": "k2c", "matrix": [2, 12], "x": 11.75, "y": 2},
+                {"label": "k2d", "matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
+
+                {"label": "k30", "matrix": [3, 0],"x": 0, "y": 3, "w": 1.25},
+                {"label": "k31", "matrix": [3, 1],"x": 1.25, "y": 3},
+                {"label": "k32", "matrix": [3, 2],"x": 2.25, "y": 3},
+                {"label": "k33", "matrix": [3, 3],"x": 3.25, "y": 3},
+                {"label": "k34", "matrix": [3, 4],"x": 4.25, "y": 3},
+                {"label": "k35", "matrix": [3, 5],"x": 5.25, "y": 3},
+                {"label": "k36", "matrix": [3, 6],"x": 6.25, "y": 3},
+                {"label": "k37", "matrix": [3, 7],"x": 7.25, "y": 3},
+                {"label": "k38", "matrix": [3, 8],"x": 8.25, "y": 3},
+                {"label": "k39", "matrix": [3, 9],"x": 9.25, "y": 3},
+                {"label": "k3a", "matrix": [3, 10],"x": 10.25, "y": 3},
+                {"label": "k3b", "matrix": [3, 11],"x": 11.25, "y": 3},
+                {"label": "k3d", "matrix": [3, 13],"x": 12.25, "y": 3, "w": 1.75},
+                {"label": "k3e", "matrix": [3, 14],"x": 14, "y": 3},
+
+                {"label": "k40", "matrix": [4, 0],"x": 0, "y": 4, "w": 1.25},
+                {"label": "k41", "matrix": [4, 1],"x": 1.25, "y": 4, "w": 1.25},
+                {"label": "k42", "matrix": [4, 2],"x": 2.5, "y": 4, "w": 1.25},
+                {"label": "k47", "matrix": [4, 7],"x": 3.75, "y": 4, "w": 6.25},
+                {"label": "k4a", "matrix": [4, 10],"x": 10, "y": 4, "w": 1.25},
+                {"label": "k4b", "matrix": [4, 11],"x": 11.25, "y": 4, "w": 1.25},
+                {"label": "k4d", "matrix": [4, 13],"x": 13, "y": 4},
+                {"label": "k4e", "matrix": [4, 14],"x": 14, "y": 4},
+                {"label": "k4f", "matrix": [4, 15],"x": 15, "y": 4}
+            ]
+        },
+        "LAYOUT_65v3_ansi_blocker": {
```

Comment:
```suggestion
        "LAYOUT_65_ansi_blocker": {
```

Reply:
This community layout LAYOUT_65_ansi_blocker is different from our layout(with a 2U blocker).

Reply:
The community layout isn't actually being used. Either way, see the [warning that was pointed out above](https://github.com/qmk/qmk_firmware/pull/20280#discussion_r1151305124). Something like `LAYOUT_blocker` & `LAYOUT_iso_blocker` would be preferable.

Reply:
> 

The community layout requires 67 keys, whereas our layout only has 65 keys. This does not compile.

Reply:
Then
> Something like `LAYOUT_blocker` & `LAYOUT_iso_blocker` would be preferable.